styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java

```diff
@@ -350,7 +350,13 @@ public void shouldGetBackfillStatusWithStatusByDefault() throws Exception {
     storage.writeEvent(SequenceEvent.create(Event.submitted(wfi, "exec-1"),                              4L, 4L));
     storage.writeEvent(SequenceEvent.create(Event.started(wfi),                                          5L, 5L));
     storage.writeActiveState(wfi, RunState.create(wfi, State.RUNNING,
-        StateData.zero(), Instant.now(), 5L));
+        StateData.newBuilder()
```

Comment:
How come this needs to be changed?

Reply:
Yeah, so let's have the test verify that they are not used.

Reply:
Addressed in later commit.